### PR TITLE
Finished

### PR DESCRIPTION
**Pull request by webmarkyn**

### CODE REVIEW

**Comment by wathigo** [comment]
**Reviewing**

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job guys :thumbsup:
- [ ] Please fix the ```stickler``` checks and submit for another review.
- [ ] Make your README more professional. Improve the formatting.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+                        <input type="email" placeholder="Your email">
+                        <button>Subscribe</button>
+                    </form>
+                </div>
+            </div>
+        </main>
+        <footer>
+            <h3 class="footer-title">BROWSE ALL SMASHING MAGAZINE TOPICS</h3>
+            <ul class="footer-taglist">
+                <li class="taglist-item" text-decoration="none"><a href="#">Performance</a></li>
```
Comment (Redvanisation) [inline index.html:319]: There is no HTML attribute such as "text-decoration", please delete it.

**Review by Redvanisation — APPROVED** [review]
### Hello @webmarkyn and @rkyankya  🌟 

### _Nice work on this project, it seems that you guys have met the requirements so I am approving your project. 👍_ 

- However, please **fix the error mentioned in the comment** I made below before submitting the project completion form.

### Project approved! you may merge to master. 🚀 

### Good luck! 🍀

**Pull request merged**